InvenTree/part/settings.py

```diff
@@ -5,34 +5,34 @@
 
 def part_assembly_default():
     """Returns the default value for the 'assembly' field of a Part object."""
-    return InvenTreeSetting.get_setting('PART_ASSEMBLY')
+    return InvenTreeSetting.get_setting('PART_ASSEMBLY', cache=False)
```

Comment:
Question: this seems like a prime candidate to cache: why is it disabled?

Reply:
Ah, good catch. I did this to work out why the tests were failing, forgot to fix it

Reply:
There is a bunch of those.